bot/test.sh

```diff
@@ -103,34 +103,34 @@ fi
 RESUME_DIR=$(grep 'Using .* as tmp directory' slurm-${SLURM_JOBID}.out | head -1 | awk '{print $2}')
 
 if [[ -z ${RESUME_DIR} ]]; then
```

Comment:
If $RESUME_DIR is empty, we have to assume that we need to create a temporary storage using `$LOCAL_TMP` which can be configured via `app.cfg`.

Reply:
I'm now doing this in either case, so both when $RESUME_DIR is set and when it's not set. Or can we point $JOB_STORAGE to $RESUME_DIR or something in case $RESUME_DIR is set?